test/controllers/slides_controller_test.rb

```diff
@@ -143,4 +143,38 @@ class SlidesControllerTest < ActionController::TestCase
 
     assert_equal "horizontal", slide.reload.orientation
   end
+
+  test "assert that `controls muted` attribute is present on slides with foreground_type: video" do
+    # Arrange
+    super_admin = users(:super_admin)
```

Comment:
Is there a reason for this variable? I don't see it used anywhere in the test but I could be missing something